api/src/main/java/javax/enterprise/inject/spi/BeanManager.java

```diff
@@ -618,4 +620,18 @@
      */
     <T> InterceptionFactory<T> createInterceptionFactory(CreationalContext<T> ctx, Class<T> clazz);
 
+
+    /**
+     *
+     * Obtains an {@link Instance} giving access to beans instances
+     *
+     * Instances of dependent scoped beans obtained with this Instance should be explicitly release with {@link Instance#destroy(Object)}
+     *
+     * @return an {@link Instance} object to request beans instances
+     * @throws IllegalStateException if called during application initialization, before the {@link AfterDeploymentValidation}
+     *         event is fired.
+     * @since 2.0
+     */
+    Instance<Object> getInstance();
```

Comment:
Just saying I don't really see the value of this issue/PR. It's only a convenient method and even as such requires you to take several steps to actually achieve the goal (get the contextual instance). And for this little (if any) gain we add yet another method to already bloated BM. But that's probably just my skepticism.